test/test_device_branch_constructors.jl

```diff
@@ -52,52 +44,228 @@ end
         )
         @test build!(op_problem_m; output_dir = mktempdir(cleanup = true)) ==
               PSI.BuildStatus.BUILT
-        # TODO: use accessors to remove the use of Symbols Directly
-        monitored_line_variable = PSI.get_variable(
-            op_problem_m.internal.optimization_container,
-            :Fp__MonitoredLine,
-        )
-        static_line_variable =
-            PSI.get_variable(op_problem_m.internal.optimization_container, :Fp__Line)
 
-        for b in monitored_line_variable
-            @test !JuMP.has_lower_bound(b)
-            @test !JuMP.has_upper_bound(b)
-        end
-        for b in static_line_variable
-            # Broken
-            #@test JuMP.has_lower_bound(b)
-            #@test JuMP.has_upper_bound(b)
-        end
+        @test check_variable_unbounded(op_problem_m, :Fp__MonitoredLine)
+        # Broken
+        # @test check_variable_bounded(op_problem_m, :Fp__MonitoredLine)
```

Comment:
Is this true or is it stale?

Reply:
Sorry that should be for static line, it does fail that test and that was a comment from the previous test code